### ISSUE TITLE
c_iter isn't used

### ISSUE THREAD

**Issue by davidADSP** [LynnHo/DCGAN-LSGAN-WGAN-WGAN-GP-Tensorflow#10]
c_iter is defined but not used in all of the WGAN files. What is the correct behaviour? i.e should the critic be optimised heavily initially or not?

Also, can you confirm that you use a learning rate parameter of 0.0002, regardless of whether RMSProp or Adam is used as the optimiser?

**Comment by LynnHo** [comment]
@davidADSP Oh, it seems to be a bug, and n_critic here should be c_iter. Thanks!
https://github.com/LynnHo/DCGAN-LSGAN-WGAN-WGAN-GP-Tensorflow/blob/f9da7801a178b048b6212283f8e6ea7bf309da8b/train_mnist_wgan.py#L104 


Besides, 0.0002 for the learning rate is the experience and different tasks should have different optimal learning rates, while I just use the same for convenience.

**Comment by davidADSP** [comment]
OK thanks! - what learning rates did you use to generate the images on the README of this repo, for the various different tasks?
